addon/components/o-s-s/layout/navbar/nav-item.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="oss-navbar-item fx-1 {{if @active "oss-navbar-item--active"}}" ...attributes>
+  <OSS::Icon @icon={{@icon}} class="font-color-gray-500" />
+  <span class="font-color-gray-600 font-size-sm">{{@label}}</span>
+  {{yield to="extra"}}
```

Comment:
our other named-block yields are often (always?) wrapped in `has-block`. I guess it's to prevent unwanted behaviours with empty content.
Can you add the condition ?
```handlebars
{{#if (has-block "extra")}}
  {{yield to="extra"}}
{{/if}}
```

Reply:
Added has-block condition wrapper around extra-content block